fcrepo-auth-common/pom.xml

```diff
@@ -63,7 +63,7 @@
       <exclusions>
         <exclusion>
           <groupId>commons-collections</groupId>
-		  <artifactId>commons-collections</artifactId>
+          <artifactId>commons-collections</artifactId>
```

Comment:
Formatting in this block still looks like it could use help:
https://github.com/peichman-umd/fcrepo4/blob/75af9201f26c1d69739ad05515090728101ef7c2/fcrepo-auth-common/pom.xml#L64-L68